### PR TITLE
Increase timeout for some imodelhub tests

### PR DESCRIPTION
**Pull request by azonys**

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

File: full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts [hunk 1]
```diff
@@ -526,20 +518,22 @@ describe("iModelHub iModelsHandler", () => {
     chai.expect(error!.errorNumber).to.be.equal(IModelHubStatus.InvalidArgumentError);
   });
 
-  it("should clone iModel", async () => {
+  it("should create iModel from another iModel", async function () {
```
Comment (austeja-bentley) [inline full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts:521]: I think the previous function syntax (() => {}) is more commonly used (at least in our tests)

File: full-stack-tests/imodelhub-client/src/imodelhub/Versions.test.ts [hunk 2]
```diff
@@ -127,7 +127,7 @@ describe("iModelHub VersionHandler", () => {
       await utils.deleteIModelByName(requestContext, contextId, imodelName);
       await utils.deleteIModelByName(requestContext, contextId, imodelName2);
       utils.getRequestBehaviorOptionsHandler().resetDefaultBehaviorOptions();
-      iModelClient.requestOptions.setCustomOptions(utils.getRequestBehaviorOptionsHandler().toCustomRequestOptions());
+      iModelClient?.requestOptions.setCustomOptions(utils.getRequestBehaviorOptionsHandler().toCustomRequestOptions());
```
Comment (austeja-bentley) [inline full-stack-tests/imodelhub-client/src/imodelhub/Versions.test.ts:130]: Why is the optional property access operator here? can iModelClient ever be undefined?

Comment (azonys) [reply full-stack-tests/imodelhub-client/src/imodelhub/Versions.test.ts:130]: Yes, it is undefined when iModel creation timeouts on "before" step

File: full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts (hunk 1 above)
Comment (azonys) [reply full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts]: Forgot to revert. This syntax is required in order to use "this". Reverted

**Pull request merged**